src/practice7_Spring/task1_SalaryEmulator/src/test/java/EmployeeServiceTest.java

```diff
@@ -0,0 +1,83 @@
+import entities.Employee;
+import entities.Position;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import services.EmployeeService;
+
+import java.util.Map;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.*;
+
+public class EmployeeServiceTest {
+
+    private EmployeeService employeeService;
+    private Position position = new Position("position");
+    private Employee employee = new Employee("firstName", "lastName", 18, position);
+    private Long maxId = Long.MAX_VALUE;
+
+    @Before
+    public void setUp() {
+        employeeService = new EmployeeService();
+    }
+
+    @After
+    public void tearDown() {
+        employeeService = null;
+        position = null;
+        employee = null;
+        maxId = null;
+    }
+
+
+    @Test
+    public void testEmployeeIsThisEmployeeHashCodeTrue() {
+        int old = employee.hashCode();
+        assertThat(employee.hashCode(), is(old));
```

Comment:
You check that employee.hashCode() is equal to employee.hashCode(). They always equal. So, this test is incorrect